### PR TITLE
Update tic-tac-toe-new-UI

### PR DESCRIPTION
**Pull request by githubparticipant**
Made updates to original file

### CODE REVIEW

**Comment by harshitashyale** [comment]
Seems like there's no score tracking logic yet?

**Comment by githubparticipant** [comment]
Yeah, I'll add that in the next release

**Comment by harshitashyale** [comment]
Also add multiplayer logic

**Comment by githubparticipant** [comment]
That's out of scope for now.

File: tic-tac-toe-new-UI [hunk 1]
```diff
@@ -2,14 +2,14 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
```
Comment (harshitashyale) [inline tic-tac-toe-new-UI:5]: This should be the other way around - (playerX, playerY)

File: tic-tac-toe-new-UI [hunk 2]
```diff
@@ -23,7 +23,7 @@ def checkwinner(playerX, playerY):
     wins = [[0, 1, 2], [3, 4, 5], [6, 7, 8],[0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
     for win in wins:
         if(sum(playerX[win[0]], playerX[win[1]], playerX[win[2]]) == 3):
-            print("Player X has won the game")
+            print("Player Y has won the game")
```
Comment (harshitashyale) [inline tic-tac-toe-new-UI:26]: this should be X

File: tic-tac-toe-new-UI [hunk 3]
```diff
@@ -53,4 +53,3 @@ if __name__ == "__main__":
             break
 
         turn = 1 - turn
```
Comment (harshitashyale) [inline tic-tac-toe-new-UI:55]:
```suggestion
        turn = turn - 1
```

**Comment by harshitashyale** [comment]
There's no sound feature?

**Comment by githubparticipant** [comment]
I'm not adding sound in this PR

**Comment by githubparticipant** [comment]
sound [feat] isn't included.

File: tic-tac-toe-new-UI (hunk 2 above)
Comment (githubparticipant) [reply tic-tac-toe-new-UI:26]: noted.

File: tic-tac-toe-new-UI (hunk 1 above)
Comment (githubparticipant) [reply tic-tac-toe-new-UI:5]: noted

File: tic-tac-toe-new-UI [hunk 4]
```diff
@@ -2,14 +2,14 @@ def sum(a, b, c ):
...
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
     zero = 'X' if playerX[0] else ('O' if playerY[0] else 0)
     one = 'X' if playerX[1] else ('O' if playerY[1] else 1)
     two = 'X' if playerX[2] else ('O' if playerY[2] else 2)
     three = 'X' if playerX[3] else ('O' if playerY[3] else 3)
     four = 'X' if playerX[4] else ('O' if playerY[4] else 4)
     five = 'X' if playerX[5] else ('O' if playerY[5] else 5)
-    six = 'X' if playerX[6] else ('O' if playerY[6] else 6)
+    six = 'X' if playerX[5] else ('O' if playerY[6] else 6)
```
Comment (harshitashyale) [inline tic-tac-toe-new-UI]:
```suggestion
    six = 'X' if playerX[6] else ('O' if playerY[6] else 6)
```

**Pull request closed without merging**